sql/src/test/java/io/crate/integrationtests/ViewsITest.java

```diff
@@ -87,6 +87,20 @@ public void testCreateViewFailsIfNameConflictsWithPartitionedTable() {
         execute("create view t1 as select 1");
     }
 
+    @Test
+    public void testCreateTableFailsIfNameConflictsWithView() {
```

Comment:
Given that the verification is already happening in the analyzer, could we turn this into an analyzer test? It's a lot cheaper than a full blown integration test.

Reply:
Converted the test.